tomni/annotation_manager/annotations/ellipse/main.py

```diff
@@ -161,12 +163,68 @@ def to_dict(self, decimals: int = 2) -> dict:
         dict_return_value = {**super_dict, **dict_ellipse}
         return dict_return_value
 
+    def to_binary_mask(self, shape: Tuple[int, int]) -> np.ndarray:
```

Comment:
dont we have this function already

Reply:
not in main and dont see a PR that does that.